sapphire/tests/corsika/test_qsub_store_corsika_data.py

```diff
@@ -1,9 +1,15 @@
 import unittest
 
+import six
+
 from mock import patch, sentinel, mock_open
 
 from sapphire.corsika import qsub_store_corsika_data
 
+if six.PY2:
```

Comment:
This is ugly and should not be necessary. I haven't found a clean solution. Needs some tinkering (or post question on SO)